### PR TITLE
Added type of selection of words with enter in seed phrase popup

### PR DESCRIPTION
**Pull request by Valentina1133**
Added second type of selection when importing seed phrase - with Enter
If all good - I can update test case https://ethstatus.testrail.net/index.php?/cases/view/703040

Screenshot 2023-10-10 at 12 45 06

### CODE REVIEW

**Comment by anastasiyaig** [comment]
@Valentina1133 it seem to be alright, but please create separate test in test rail for this

**Review by anastasiyaig — APPROVED** [review]

File: gui/screens/onboarding.py [hunk 1]
```diff
@@ -211,7 +211,15 @@ def input_seed_phrase(self, seed_phrase: typing.List[str]):
...
         for index, word in enumerate(seed_phrase, start=1):
             self._seed_phrase_input_text_edit.real_name['objectName'] = f'statusSeedPhraseInputField{index}'
-            self._seed_phrase_input_text_edit.text = word
+            if type_of_selection == 'auto':
+                self._seed_phrase_input_text_edit.text = word
+            elif type_of_selection == 'enter':
+                word_letters = []
+                for letter in word:
+                    word_letters.append(letter)
+                word_to_put = ''.join(word_letters[:-1])
```
Comment (StateOf-Vlado) [inline gui/screens/onboarding.py:220]: Valentina, the string is a list of chars you don't need to convert it into the list
just use `word[:-1]`

File: tests/onboarding/test_onboarding_import_seed.py [hunk 2]
```diff
@@ -23,11 +23,15 @@ def keys_screen(main_window) -> KeysView:
 
 @allure.testcase('https://ethstatus.testrail.net/index.php?/cases/view/703040', 'Import: 12 word seed phrase')
 @pytest.mark.case(703040)
-@pytest.mark.parametrize('user_account', [constants.user.user_account_two])
-def test_import_seed_phrase(aut: AUT, keys_screen, main_window, user_account):
+@pytest.mark.parametrize('user_account', [constants.user.user_account_one])
+@pytest.mark.parametrize('type_of_selection', [
```
Comment (StateOf-Vlado) [inline tests/onboarding/test_onboarding_import_seed.py:27]: please rename type_of_selection to autocomplete
As I understand you verify the correct autocomplete here

File: tests/onboarding/test_onboarding_import_seed.py [hunk 3]
```diff
@@ -23,11 +23,15 @@ def keys_screen(main_window) -> KeysView:
...
+    pytest.param('auto'),
+    pytest.param('enter')
```
Comment (StateOf-Vlado) [inline tests/onboarding/test_onboarding_import_seed.py:29]: please change string values that you are using as boolean to boolean
like 'auto' and 'enter'

**Review by StateOf-Vlado — CHANGES_REQUESTED** [review]

File: gui/screens/onboarding.py (hunk 1 above)
Comment (Valentina1133) [reply gui/screens/onboarding.py]: God, feel that I'm stupid :( Fixed

File: tests/onboarding/test_onboarding_import_seed.py (hunk 2 above)
Comment (Valentina1133) [reply tests/onboarding/test_onboarding_import_seed.py]: Done

File: tests/onboarding/test_onboarding_import_seed.py (hunk 3 above)
Comment (Valentina1133) [reply tests/onboarding/test_onboarding_import_seed.py]: Please, check, not sure if I got what you mean, but I changed to autocomplete with params True and False

File: gui/screens/onboarding.py [hunk 4]
```diff
@@ -211,7 +211,12 @@ def input_seed_phrase(self, seed_phrase: typing.List[str]):
 
         for index, word in enumerate(seed_phrase, start=1):
             self._seed_phrase_input_text_edit.real_name['objectName'] = f'statusSeedPhraseInputField{index}'
-            self._seed_phrase_input_text_edit.text = word
+            if autocomplete is False:
+                self._seed_phrase_input_text_edit.text = word
+            elif autocomplete is True:
+                word_to_put = word[:-1]
+                self._seed_phrase_input_text_edit.text = word_to_put
+                driver.type(self._seed_phrase_input_text_edit.object, "<Return>")
```
Comment (StateOf-Vlado) [inline gui/screens/onboarding.py:219]:
```
                if autocomplete:
                      word_to_put = word[:-1]
                      self._seed_phrase_input_text_edit.text = word_to_put
                      driver.type(self._seed_phrase_input_text_edit.object, "<Return>")
                else:
                      self._seed_phrase_input_text_edit.text = word
```

**Review by StateOf-Vlado — APPROVED** [review]
simple fix recommendation added

File: gui/screens/onboarding.py (hunk 4 above)
Comment (Valentina1133) [reply gui/screens/onboarding.py:217]: Fixed

**Pull request merged**